samples/install-without-bom/pom.xml

```diff
@@ -147,6 +147,8 @@
         <artifactId>maven-failsafe-plugin</artifactId>
         <version>3.5.2</version>
         <configuration>
+          <forkCount>10</forkCount>
+          <reuseForks>false</reuseForks>
```

Comment:
I don't think this does anything at the moment, as there are no integration tests defined in this project. Are we planning on adding tests here?

Reply:
It is. Currently we are runnings tests three times in all three modules.